spec.html

```diff
@@ -39904,6 +39904,57 @@ <h1 id="ao-issuperreference">IsSuperReference ( _V_ )</h1>
         1. Return ? ForIn/OfBodyEvaluation(|BindingIdentifier|, |Statement|, _keyResult_, ~enumerate~, ~varBinding~, _labelSet_).
       </emu-alg>
     </emu-annex>
+
+    <emu-annex id="sec-html-document-dot-all-exotic-object">
+      <h1>HTML Document Dot All Exotic Object</h1>
+      <p>An <dfn>HTML document dot all exotic object</dfn> is an exotic object that behaves like undefined in the <emu-xref href="#sec-toboolean">ToBoolean</emu-xref> and <emu-xref href="#sec-abstract-equality-comparison">Abstract Equality Comparison</emu-xref> abstract operations and when used as an operand for the <emu-xref href="#sec-typeof-operator">typeof operator</emu-xref>. HTML document dot all exotic objects have all the ordinary object internal methods and slots.</p>
```

Comment:
> typeof operator

It looks like `typeof` should be `code`ified

> HTML document dot all exotic objects have all the ordinary object internal methods and slots

This isn't quite right, as HTML will define one with overridden internal methods. Not sure what the right phrasing is... "By default, ... (but other specifications may create HTML document dot all exotic objects with overridden internal methods)"?